src/main/kotlin/org/radarbase/redcap/util/RedCapTrigger.kt

```diff
@@ -213,6 +216,7 @@ class RedCapTrigger(value: String) {
     }
 
     companion object {
+        private val LOGGER = LoggerFactory.getLogger(RedCapClient::class.java)
```

Comment:
Why not use this class' name in the getLogger

Reply:
Ah I missed this when I copied it. Updated now.